tox.ini

```diff
@@ -7,7 +7,7 @@ deps =
     pytest-cov
     scrapy >= 2.1.0
     pytest-twisted
-    web-poet
+    web-poet @ git+https://git@github.com/scrapinghub/web-poet@handle_urls#egg=web-poet
```

Comment:
Reminder to update this when https://github.com/scrapinghub/web-poet/pull/16 is released.